stackstate_checks_base/stackstate_checks/base/data/agent_requirements.in

```diff
@@ -11,6 +11,7 @@ pymysql==0.8.0
 pywin32==224; sys_platform == 'win32'
 pyyaml==3.13
 requests==2.20.1
+requests_ntlm==1.1.0
```

Comment:
I believe it's not needed here but putting inside the requirements of the check is fine.